app/controllers/schedules_controller.rb

```diff
@@ -1,7 +1,8 @@
 class SchedulesController < ApplicationController
+  load_and_authorize_resource
   protect_from_forgery with: :null_session
   before_action :respond_to_options
-  load_and_authorize_resource :conference, find_by: :short_title
```

Comment:
Following are the reasons to change
`load_and_authorize_resource :conference, find_by: :short_title` to `load_resource :conference, find_by: :short_title`

- Keeping `load_and_authorize_resource :conference, find_by: :short_title` doesn't allow user to access schedule without login because we don't have can ability for show action of conference for `not_signed_in` user 
- In order to allow unsigned user to view schedule which indirectly requires conference, I have removed `authorize_resource` part and added authorization for `schedules_controller` to make up for it.